scan-command/src/test/java/io/ballerina/scan/internal/Rule005Test.java

```diff
@@ -0,0 +1,35 @@
+package io.ballerina.scan.internal;
+
+import io.ballerina.projects.Document;
+import io.ballerina.scan.Issue;
+import io.ballerina.scan.RuleKind;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Rule005Test extends StaticCodeAnalyzerTest {
+    private static final String UNUSED_CLASS_PRIVATE_FIELDS = "Unused class private fields";
+
+    @Test(description = "test checkpanic analyzer")
+    void testUnusedClassFieldsAnalyzer() {
```

Comment:
Needs to reflect "private".

Reply:
Using default qualifier is the way to write test functions

Reply:
That's not what I meant. I meant that it needs to say something like unused**Private**Fields.